src/main/groovy/com/sequenceiq/ambari/client/services/ViewService.groovy

```diff
@@ -0,0 +1,80 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.sequenceiq.ambari.client.services
+
+import groovy.json.JsonBuilder
+import groovy.util.logging.Slf4j
+
+@Slf4j
+trait ViewService extends ClusterService {
```

Comment:
I created a method which accepts parameters for different views, but the 3 default remains as it takes the burden from the caller to know all the properties.